processors/github/github_processor_phase1_per_repo.py

```
@@ -0,0 +1,721 @@
+    
+    def _get_latest_commit(self, repo_path: Path) -> str:
+        """Get latest commit hash."""
+        try:
+            result = subprocess.run(
+                ['git', 'rev-parse', 'HEAD'],
+                cwd=repo_path,
+                capture_output=True,
+                text=True
+            )
+            return result.stdout.strip()[:12] if result.returncode == 0 else 'unknown'
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)